lib/contractor_list/domain/models/contractor.dart

```diff
@@ -0,0 +1,36 @@
+
+import 'package:equatable/equatable.dart';
+import 'package:rate_my_contractor/contractor_list/data/models/license_dto.dart';
+import '../../data/models/contractor_dto.dart';
+class Contractor extends Equatable  {
+  /// Everything that a contractor DTO would have
+  /// Plus the list of Licenses associated with that contractor
+  final String id;
+  final String companyName;
+  final String address;
+  final String? ownerName;
+  final String? image;
+  final double? rating;
```

Comment:
We should not show a null value for stars. Think about perhaps adding a default value instead. If we don't get ratings we should just default to 0 or something.

Reply:
An alternative is checking in the UI if null. If it's null then instead of showing the start show some text like "No Ratings" or something like that. Up to you guys